content/v2/openapi.yml

```
@@ -3861,6 +3861,14 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/NameServer'
+    EventDomainRegistrantStarted:
```

I think there is a typo here:

```suggestion
    EventDomainRegistrantChangeStarted:
```